app/mailers/notification_mailer.rb

```diff
@@ -0,0 +1,19 @@
+class NotificationMailer < ApplicationMailer
+  layout 'notifications/email'
+
+  # Send email using input template URL and objects
+  #
+  # @param recipient [User] the recipient of this email
+  # @param sender [Object] the one who sends out this email
```

Comment:
Actually if you don't care about the type, just omit the type declaration.

Your parameter descriptions are still not complete sentences.